### PR TITLE
Fix and Workarounds for BigQueryHook

### PR DESCRIPTION
**Pull request by csmc88**
#### What does this PR do?
Adds a workaround for two errors found in the BigQueryHook class. 
1) A missed call to super to initialize the parent class in `BigQueryPandasConnector`
2) A workaround when `_parse_data` from `pandas_gbq` fails to parse a result with a single column

#### What are the relevant JIRA tickets?
Workaround in Airflow for [pandas-gbq issue 114](https://github.com/pydata/pandas-gbq/issues/114)

#### Where should the reviewer start?
Look at `airflow/contrib/hooks/bigquery_hook.py`

### CODE REVIEW

File: airflow/contrib/hooks/bigquery_hook.py [hunk 1]
```diff
@@ -138,7 +138,10 @@ class BigQueryPandasConnector(GbqConnector):
     without forcing a three legged OAuth connection. Instead, we can inject
     service account credentials into the binding.
     """
-    def __init__(self, project_id, service, reauth=False, verbose=False, dialect='legacy'):
+    def __init__(self, project_id, service, reauth=False, verbose=False, dialect='legacy', **kwargs):
```
Comment (zubieta) [inline airflow/contrib/hooks/bigquery_hook.py:168]: Maybe we can put a comment pointing to the relevant JIRA issue, so we can check when it is fixed.

And probably this commit should be a different PR, so it is easy to reset the patches separately when their are fixed upstream.

**Review by zubieta — CHANGES_REQUESTED** [review]

File: airflow/contrib/hooks/bigquery_hook.py (hunk 1 above)
Comment (csmc88) [reply airflow/contrib/hooks/bigquery_hook.py:168]: I'm closing this PR as this has been split into two PRs

[Fix Attribute Error PR](https://github.com/wizeline/incubator-airflow/pull/4)

and  

[Workaround for Pandas GBQ](https://github.com/wizeline/incubator-airflow/pull/5)
